source/models/the-fixture-adapter.md

```diff
@@ -79,6 +79,6 @@ a uniquely identifiable field. By default, Ember Data assumes this key
 is called `id`. Should you not provide an `id` field in your fixtures, or
 not override the primary key, the Fixture Adapter will throw an error.
 
-[1]: /guides/models/defining-models
```

Comment:
I believe they should be `../defining-models`, but you'll have to test to confirm...